packages/platform-sdk-atom/src/services/message.ts

```diff
@@ -22,7 +22,7 @@ export class MessageService implements Contracts.MessageService {
 
 	public async sign(input: Contracts.MessageInput): Promise<Contracts.SignedMessage> {
 		try {
-			const { publicKey, privateKey } = await this.#identityService.keys().fromMnemonic(input.mnemonic);
+			const { publicKey, privateKey } = await this.#identityService.keys().fromMnemonic(input.mnemonic!);
```

Comment:
This needs a proper check that the mnemonic is set now that it isn't enforced by TypeScript.